example/utils/performance.profiler.ts

```diff
@@ -0,0 +1,25 @@
+import { Logger } from '@nestjs/common';
+
+export class PerformanceProfiler {
+  started: number;
+  description: string;
+
+  stopped: number = 0;
+  duration: number = 0;
+
+  constructor(description: string = '') {
+    this.started = Date.now();
```

Comment:
If precision is important, you might want to consider using performance.now() over Date.now()

Reply:
not really important for the example app